### PR TITLE
Update foreign-types to 0.3

### PR DESCRIPTION
**Pull request by nox**

### CODE REVIEW

File: openssl/src/stack.rs [hunk 1]
```diff
@@ -72,12 +71,6 @@ impl<T: Stackable> AsRef<StackRef<T>> for Stack<T> {
     }
 }
 
-impl<T: Stackable> Borrow<StackRef<T>> for Stack<T> {
```
Comment (sfackler) [inline openssl/src/stack.rs]: `Stack<T>` doesn't have a `foreign_type!`-generated impl, so isn't this still necessary?

Comment (nox) [reply openssl/src/stack.rs]: Oops, restored.

**Comment by sfackler** [comment]
Thanks!

**Comment by nox** [comment]
This is a breaking change btw.

**Comment by sfackler** [comment]
Oh right, derp. Will revert for now, and re-land some time in the next month or so when 0.10 is ready to land.

**Pull request merged**